[RHACS] Added a note auto-generated registries are disabled in ACS (#79031)
openshift/openshift-docs: modules/automatic-configuration-image-registry.adoc

// * integration/integrate-with-image-registries.adoc
:_mod-docs-content-type: CONCEPT
[id="automatic-configuration-image-registry_{context}"]
= Automatic configuration

[role="_abstract"]
{product-title} includes default integrations with standard registries, such as Docker Hub and others. It can also automatically configure integrations based on artifacts found in the monitored clusters, such as image pull secrets. Usually, you do not need to configure registry integrations manually.

[IMPORTANT]
====

> * If you are using a GCR registry, {product-title} does not create a registry integration automatically.
> * If you are using {product-title-managed}, automatic configuration is not available. You must create registry integrations manually.

Reviewer: ```suggestion
* If you use a Google Container Registry (GCR), {product-title} does not create a registry integration automatically.
* If you use {product-title-managed}, you must create a registry integration manually.
```

Author: Thank you @shipsing for checking this one. I've updated the first bullet point. However, for the second one, I want to highlight that the feature `Automatic configuration` is not available in `product-title-managed`. Therefore, I've stated the name. I've updated the sentence to the following:
```
If you use {product-title-managed}, automatic configuration is unavailable, 
and you must manually create registry integrations.
```
